source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md

```diff
@@ -1,25 +1,40 @@
 ---
 title: Understanding Guest Agents and Other Tools
-authors: bobdrad, gianluca, nkesick, sven, vfeenstr
+authors: bobdrad, gianluca, nkesick, sven, vfeenstr, sandrobonazzola
 ---
 
-<!-- TODO: Content review -->
 
 # Understanding Guest Agents and Other Tools
 
 ## Introduction
 
 There are three tools which together can provide the best experience when using a virtual machine with oVirt.
 
-*   oVirt Guest Agent - Recommended
+*   QEMU Guest Agent - Recommended
 *   VirtIO Drivers - Recommended
 *   Spice Drivers - Recommended if you use the Spice console
 
+There used to be also:
+
+*   oVirt Guest Agent - Deprecated and not available on recent OSs
+
+Please refer to [Virtual Machine Management Guide](/documentation/vmm-guide/Virtual_Machine_Management_Guide.html) for agents and drivers installation instructions.
+
+
 ## Agents
 
 ### oVirt Guest Agent
 
-The oVirt Guest Agent provides information, notifications, and actions between the oVirt web interface and the guest. A more detailed description can be found at [Guest Agent](guest-agent). The agent provides the Machine Name, Operating System, IP Addresses, Installed Applications, Network and RAM usage and others details to the web interface. The agent also provides Single Sign On so a authenticated user to the web interface does not need to authenticate again when connected to a VM.
+**The oVirt Guest Agent has been deprecated** in favor of QEMU Guest Agent and is not available anymore for recent OS. It can still be found for less recent OS in their own distribution repositories.
+The oVirt Guest Agent provides information, notifications, and actions between the oVirt web interface and the guest. A more detailed description can be found at [oVirt Guest Agent](guest-agent.html).
+The agent provides the Machine Name, Operating System, IP Addresses, Installed Applications, Network and RAM usage and others details to the web interface. The agent also provides Single Sign On 
+so a authenticated user to the web interface does not need to authenticate again when connected to a VM.
+
+### QEMU Guest Agent
+
+The QEMU Guest Agent helps management applications with executing functions which need assistance of the guest OS. For example, freezing and thawing filesystems, entering suspend.
+It provides information, notifications, and actions between the oVirt web interface and the guest. A more detailed description can be found at [QEMU Guest Agent](https://wiki.libvirt.org/page/Qemu_guest_agent).
```

Comment:
maybe worth putting it before the oVirt Guest Agent?

Reply:
moved